README.md

```diff
@@ -68,4 +68,33 @@ There is always a lot of work to do, and already well over 150 rules to maintain
 -   Open new issues about your ideas for making stylelint better, and pull requests to show us how your idea works.
 -   Create or contribute to ecosystem tools, like the plugins for [Atom](https://github.com/AtomLinter/linter-stylelint) and [Sublime Text](https://github.com/kungfusheep/SublimeLinter-contrib-stylelint).
 
+## Semantic Versioning Policy
+
+stylelint follows [semantic versioning](http://semver.org). However, due to the nature of stylelint as a code quality tool, it's not always clear when a minor or major version bump occurs. To help clarify this for everyone, we've defined the following semantic versioning policy for stylelint:
+
+-   Patch release (intended to not break your lint build)
+    -   A bug fix in a rule that results in stylelint reporting fewer errors.
+    -   A bug fix to the CLI or core (including formatters).
+    -   Improvements to documentation.
+    -   Non-user-facing changes such as refactoring code, adding, deleting, or modifying tests, and increasing test coverage.
+    -   Re-releasing after a failed release (i.e., publishing a release that doesn't work for anyone).
+
+-   Minor release (might break your lint build)
+    -   A bug fix in a rule that results in stylelint reporting more errors.
+    -   A new rule is created.
+    -   A new option to an existing rule that does not result in stylelint reporting more errors by default.
+    -   An existing rule is deprecated.
+    -   A new CLI capability is created.
+    -   New capabilities to the public API are added (new classes, new methods, new arguments to existing methods, etc.).
+    -   A new formatter is created.
+
+-   Major release (likely to break your lint build)
+    -   [`stylelint-config=standard`](https://github.com/stylelint/stylelint-config-standard) is updated.
+    -   A new option to an existing rule that results in stylelint reporting more errors by default.
```

Comment:
Our options are opt-in and so I don't think this will ever happen. How's about the following instead?:

"A change in the documented behaviour of an existing rule results in stylelint reporting more errors by default."

The use of "*documented* behaviour" is too distinguish this from *bug fixes* that result in more errors.